workload/scripts/workload-install.ps1

```diff
@@ -110,6 +114,26 @@ if (-Not $(Test-Path "$DotnetInstallDir")) {
     Write-Error "No installed dotnet '$DotnetInstallDir'."
 }
 
+# Check installed dotnet version
+$DotnetCommand = "$DotnetInstallDir\dotnet"
+if (Get-Command $DotnetCommand -ErrorAction SilentlyContinue)
+{
+    $DotnetVersion = Invoke-Expression "& '$DotnetCommand' --version"
```

Comment:
```suggestion
    [Version]$DotnetVersion = Invoke-Expression "& '$DotnetCommand' --version"
```
you can use `System.Version` class instead of `string`.
for example
```powershell
PS C:\Users\wy80.choi> [Version]$dotnetVersion = Invoke-Expression "& dotnet --version"
PS C:\Users\wy80.choi> $dotnetVersion

Major  Minor  Build  Revision
-----  -----  -----  --------
6      0      101    -1


PS C:\Users\wy80.choi> $dotnetVersion.Major
6
```

Reply:
Looks very nice to use `Version` class. I will update it. Thanks

Reply:
Just tried to apply `System.Version`, however, it does not take all dotnet version as  a valid version format.
For example. there could also be `preview`, `rc`, or etcs like `5.0.100-preview.8`, `6.0.100-rc.1.`
I could use `System.Version` after split the version with `-` symbol, but not sure if it's more efficient.

Reply:
I got it. we can find the better solution later. 😄